mappings/net/minecraft/text/component/KeybindComponent.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/unmapped/C_scqpkchq net/minecraft/text/component/KeybindComponent
```

Comment:
Don't we use text instead of component?

Reply:
These are objects that turn into `Mutable` text objects